### PR TITLE
Feature/472 eoc log

### LINKED ISSUE #391: Typo in the state
The property [`ecoLog`](https://github.com/hpi-sam/digital-fuesim-manv/blob/dev/shared/src/state.ts#L43) in the state should rather be named `eocLog` as it's the `Emergency Operation Center Log`.

@hpistudent72 This might be possible to fix when doing #384 or its successor which introduces the logging functionality.

### LINKED ISSUE #472: Add Emergency Operation Center Log

### PR DESCRIPTION
**Pull request by ClFeSc**
~~Based on #476~~

Closes #472 
Closes #391

### CODE REVIEW

**Comment by ClFeSc** [comment]
`Einsatztagebuch` should be the correct name.

**Comment by Dassderdie** [comment]
@ClFeSc I changed the UI a bit.
image

File: backend/src/database/state-migrations/3-update-eoc-log.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ * @param entry The {@link EocLogEntry} for version 2
+ */
+function convertEocLogEntryInPlace(
+    entry: object,
+    clients: { [key: UUID]: Client }
+): EocLogEntry {
+    interface EocLogEntry3 {
+        id: UUID;
+        timestamp: ImmutableDate;
```
Comment (Dassderdie) [inline backend/src/database/state-migrations/3-update-eoc-log.ts:41]: TODO:

File: frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts [hunk 2]
```diff
@@ -98,6 +103,13 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
                     color: 'success',
                 });
             }
+            this.apiService.proposeAction({
+                type: '[Emergency Operation Center] Add Log Entry',
+                message: `Alarmgruppe ${alarmGroup.name} wurde alarmiert zu ${targetTransferPoint.externalName}!`,
```
Comment (Dassderdie) [inline frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts:108]: TODO: I don't like the sound of this message yet...
Maybe: 

```ts
`${alarmGroup.name} wurde nach ${targetTransferPoint.externalName} alarmiert!`
```

Comment (ClFeSc) [reply frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts:108]: @christianzoellner

**Review by Dassderdie — APPROVED** [review]
@ClFeSc have a look on my changes

File: frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts (hunk 2 above)
Comment (Dassderdie) [reply frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts:108]: We will merge this soon - I would create another PR for changing this message.

**Pull request merged**